oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/index/ElasticBulkProcessorHandler.java

```diff
@@ -151,6 +152,7 @@ private BulkProcessor initBulkProcessor() {
         return BulkProcessor.builder(requestConsumer(),
                 new OakBulkProcessorListener(), this.indexName + "-bulk-processor")
                 .setBulkActions(indexDefinition.bulkActions)
+                .setConcurrentRequests(BULK_PROCESSOR_CONCURRENCY)
```

Comment:
I am wondering if it's ok to set this concurrency to 4 as default here ? The intended change is only for OutOfBandIndexing via oak-run.
If I am not wrong this change here would impact the async indexing as well. 
@fabriziofortino  ?

Reply:
I agree. The concurrent number of requests should be configurable (1 by default).